### PR TITLE
Customization nodejs version for travis

### PR DESCRIPTION
**Pull request by AlekseyLeshko**
Now you can be entered versions of nodejs #201

### CODE REVIEW

File: generators/app/index.js [hunk 1]
```diff
@@ -301,6 +316,11 @@ module.exports = class extends Generator {
   default() {
     if (this.options.travis) {
       let options = { config: {} };
+
+      if (this.props.nodejsVersion) {
+        options.config.node_js = this.props.nodejsVersion.split(/\s*,\s*/g); // eslint-disable-line camelcase
```
Comment (SBoudrias) [inline generators/app/index.js]:
```suggestion
        // eslint-disable-next-line camelcase
        options.config.node_js = this.props.nodejsVersion.split(',');
```

File: generators/app/index.js [hunk 2]
```diff
@@ -226,6 +226,20 @@ module.exports = class extends Generator {
     });
   }
 
+  _askForTravis() {
+    const prompts = [
+      {
+        name: 'nodejsVersion',
+        message: 'Enter nodejs version (comma to split)',
```
Comment (SBoudrias) [inline generators/app/index.js]: > Enter Node versions (comma separated)

File: __tests__/app.js [hunk 3]
```diff
@@ -30,7 +30,8 @@ describe('node:app', () => {
         authorEmail: 'hi@yeoman.io',
         authorUrl: 'http://yeoman.io',
         keywords: ['foo', 'bar'],
-        includeCoveralls: true
+        includeCoveralls: true,
+        nodejsVersion: 'v10.4.1,v10'
```
Comment (SBoudrias) [inline __tests__/app.js]: Let's just use `node` instead of `nodejs` for all the namings. (there's a few other spot to also update)

**Comment by AlekseyLeshko** [comment]
I fixed remarks, check again, please

**Comment by SBoudrias** [comment]
Thanks!

**Pull request merged**